frontend/src/metabase/plugins/index.ts

```diff
@@ -586,7 +586,7 @@ export interface SimpleDataPickerProps {
   isInitiallyOpen: boolean;
   triggerElement: ReactNode;
   setSourceTableFn: (tableId: TableId) => void;
-  entityTypes: EmbeddingEntityType[];
+  entityTypes: ModularEmbeddingEntityType[];
```

Comment:
On master, the change was on `frontend/src/metabase/plugins/oss/embedding.ts`, so there was a conflict, and that was it.

The other 2 conflicts have import conflicts, and the fix is just to remove unrelated imports.